vscode-car-plugin/src/main/java/org/wso2/maven/CAppHandler.java

```diff
@@ -939,14 +939,17 @@ public boolean accept(File dir, String name) {
 
             for (File libFile : libFiles) {
                 String fileNameWithoutExt = libFile.getName().substring(0, libFile.getName().length() - 4);
-
-                // Add dependency to the list
+                
+                // Create a unique name for the dependency by appending connector name as a suffix
+                String uniqueDependencyName = fileNameWithoutExt + "_" + connectorName;
```

Comment:
Is it better to start with the connector name instead?

Reply:
fixed